<?xml version="1.0"?>
<!-- Runs scan_finer with a fined_scan publisher. -->
<launch>
  <node pkg="scan_fining" type="scan_fining" name="scan_fining" output="screen">
    <param name="world_frame"     type="string" value="map"/>
    <param name="scanner_frame"   type="string" value="base_scan"/>

    <param name="scan_name"           type="string" value="scan"/>
    <param name="costmap_name"        type="string" value="costmap_node/costmap/costmap"/>
    <param name="output_name"         type="string" value="fined_scan"/>

    <param name="max_scanner_range"   type="double" value="3.5"/>
  </node>
</launch>
